crypto/pkcs11key/key_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !nopkcs11
+// +build pkcs11
```

Comment:
Needs a build tag. key.go and key_test.go make use of miekg/pkcs11 data structures.